tests/unit/maxmemory.tcl

```diff
@@ -241,3 +241,22 @@ test_slave_buffers {slave buffer are counted correctly} 1000000 10 0 1
 # test again with fewer (and bigger) commands without pipeline, but with eviction
 test_slave_buffers "replica buffer don't induce eviction" 100000 100 1 0
 
+start_server {tags {"maxmemory"}} {
+    test {Can't rehash if used memory exceeds maxmemory after rehashing} {
```

Comment:
i'm no expert to English, but i think you're confusing "don't" and "can't"
in that case let's rename the PR title too (will set the commit comment later on)
```suggestion
    test {Don't rehash if used memory exceeds maxmemory after rehashing} {
```